### PR TITLE
[1.1.x] G33 magic numbers

### PR DESCRIPTION
**Pull request by LVD-AC**
see remarks in #8137

### CODE REVIEW

File: Marlin/enum.h [hunk 1]
```diff
@@ -58,6 +58,22 @@ enum AxisEnum {
...
+  _4P_STEP = _7P_STEP * 2,   // 4-point step
+  CEN      = 0,              // the 7 main calibration points - add definitions if needed
+  __A      = 1,
+  _AB      = __A + _7P_STEP,
+  __B      = _AB + _7P_STEP,
+  _BC      = __B + _7P_STEP,
+  __C      = _BC + _7P_STEP,
+  _CA      = __C + _7P_STEP,
+  NPP      = _7P_STEP * 6    // number of calibration points on the radius
+};
```
Comment (thinkyhead) [inline Marlin/enum.h]: It's not really the proper use of `enum`. I would recommend doing these as `constexpr uint8_t` instead.

Comment (thinkyhead) [reply Marlin/enum.h]: As these are only used by `G33`, they would be better defined just above the `G33` function in the .cpp file.

**Review by thinkyhead — CHANGES_REQUESTED** [review]
Overall looks good. Just a couple tweaks requested. Same will apply to the 2.0.x version.

File: Marlin/enum.h (hunk 1 above)
Comment (LVD-AC) [reply Marlin/enum.h]: Reworked this

**Comment by LVD-AC** [comment]
doc is changed in #89 to reflect extra probe grids

**Review by thinkyhead — APPROVED** [review]

**Pull request merged**